docs/azure_arc_jumpstart/azure_arc_servers/day2/arc_extended_security_updates/_index.md

```diff
@@ -0,0 +1,173 @@
+
+    > **NOTE: The Jumpstart scenarios are designed with as much ease of use in-mind and adhering to security-related best practices whenever possible. It is optional but highly recommended to scope the service principal to a specific [Azure subscription and resource group](https://docs.microsoft.com/cli/azure/ad/sp?view=azure-cli-latest) as well considering using a [less privileged service principal account](https://docs.microsoft.com/azure/role-based-access-control/best-practices)**
+
+## Deployment Options and Automation Flow
+
+For you to get familiar with the automation and deployment flow, below is an explanation.
+
+- User provides the Bicep template parameter values, either via the portal or editing the parameters file. These parameters values are used throughout the deployment.
+
+- User will run the Bicep template at resource group level.
+
+- User logs in to the Azure VM using Azure Bastion or RDP to trigger the Logon script. The script will:
+  - Install Hyper-V
+  - Create Windows and/or SQL VMs in Hyper-V
+  - Onboard the Hyper-V VMs as Arc-enabled resources
+
+## Bicep template deployment
+
+- Clone the Azure Arc Jumpstart repository
+
+    ```shell
+    git clone https://github.com/microsoft/azure_arc.git
+    ```
+
+- Edit the parameter file to match your environment. You will need to provide:
+  - _`spnClientId`_: the AppId of the service principal you created before.
+  - _`spnClientSecret`_the password of the service principal you created before.
+  - _`spnTenantId`_: your Azure AD's tenant ID.
+  - _`windowsAdminUsername`_: Windows admin username for your Azure VM.
+  - _`windowsAdminPassword`_: password for the Windows admin username.
+  - _`deployBastion`_: whether or not you'd like to deploy Azure Bastion to access the Azure VM. Values can be "true" or "false"
+  - _`esu`_: this parameter will allow you to control what VMs will be deployed. It can be`_:
+    - _`ws`_: to only deploy a Windows Server 2012 VM that will be registered as an Arc-enabled server.
+    - _`sql`_: to only deploy a SQL Server 2012 VM that will be registered as an Arc-enabled SQL server.
+    - _`both`_: to deploy both a Windows Server 2012 VM and a SQL Server 2012 VM that will be Arc enabled.
+
+  ![Parameters file](./01.png)
+
+- To run the automation, navigate to the [deployment folder](https://github.com/microsoft/azure_arc/tree/main/azure_arc_servers_jumpstart/esu/bicep). From the deployment folder run the below command:
+
+  ```shell
+    az group create --name <your_resource_group> --location <your_azure_region>
+    az deployment group create --resource-group <your_resource_group> main.bicep --parameters main.parameters.json
+  ```
+
+    For example:
+
+  ```shell
+    az group create --name Arc-ESU-Demo --location "westeurope"
+    az deployment group create --resource-group Arc-ESU-Demo --name Arc-ESU-Demo --template-file main.bicep --parameters main.parameters.jsonn
+  ```
```

Comment:
Please update to multiline commands like in other scenarios for ease of readability.